### PR TITLE
csi: add a parameter to modify the maxUnavailable value for cephfs ds

### PR DESCRIPTION
**Pull request by scervantes-stratio**
**Description of your changes:**
The default value of maxUnavailable of daemonset is 1.
Sometimes the upgrade can be slow due to this setting.
I add a parameter to change the value for CephFS daemonset.

**Which issue is resolved by this Pull Request:**
None

**Checklist:**

- [x] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure)).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [x] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [x] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: Documentation/Helm-Charts/operator-chart.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ The following table lists the configurable parameters of the rook-operator chart
 | `csi.attacher.image` | Kubernetes CSI Attacher image | `registry.k8s.io/sig-storage/csi-attacher:v4.1.0` |
 | `csi.cephFSFSGroupPolicy` | Policy for modifying a volume's ownership or permissions when the CephFS PVC is being mounted. supported values are documented at https://kubernetes-csi.github.io/docs/support-fsgroup.html | `"File"` |
 | `csi.cephFSPluginUpdateStrategy` | CSI CephFS plugin daemonset update strategy, supported values are OnDelete and RollingUpdate | `RollingUpdate` |
+| `csi.cephFSPluginUpdateStrategyMaxUnavailable` | A maxUnavailable parameter of CSI CephFS plugin daemonset update strategy. | `1` |
```
Comment (parth-gr) [inline Documentation/Helm-Charts/operator-chart.md:59]: If you need to edit this file, 
You also need to edit the `Edit the corresponding *.gotmpl.md file instead
`

**Review by Madhu-1 — COMMENTED** [review]
can you please do the same change for NFS and RBD also?

File: Documentation/Helm-Charts/operator-chart.md (hunk 1 above)
Comment (scervantes-stratio) [reply Documentation/Helm-Charts/operator-chart.md]: I restored the file.
Thank you.

**Comment by scervantes-stratio** [comment]
> can you please do the same change for NFS and RBD also?

The change for RBD was done here: [https://github.com/rook/rook/commit/ac4b47d71a9ae5e605e02d31532bff20580d5a4f](url)

I made the change for NFS

**Comment by travisn** [comment]
Please see the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/) to address the CI failures for commitlint, DCO, and docs-check.

**Comment by scervantes-stratio** [comment]
> Please see the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/) to address the CI failures for commitlint, DCO, and docs-check.

done, thank you

**Comment by subhamkrai** [comment]
> > Please see the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/) to address the CI failures for commitlint, DCO, and docs-check.
> 
> done, thank you

still commitlint and docs-check CI are failing

**Comment by travisn** [comment]
@scervantes-stratio Are you planning on making the updates? Thanks!

**Pull request closed without merging**